src/ServiceManagerBase.sol

```diff
@@ -135,6 +137,114 @@ abstract contract ServiceManagerBase is ServiceManagerBaseStorage {
         _setRewardsInitiator(newRewardsInitiator);
     }
 
+    function migrateOperatorSetIds(uint32[] memory operatorSetsToCreate) external onlyOwner{
```

Comment:
Nit: `migrateAndCreateOperatorSetIds` ? Function does two things -> becomes an operator set avs and creates operator set Ids

Reply:
[39d56b1](https://github.com/Layr-Labs/eigenlayer-middleware/pull/286/commits/39d56b1996e38ea7d6437b9c83e566909c5b1d23)